extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -109,6 +109,28 @@ export class ConsentPolicyManager {
     });
   }
 
+  /**
+   * Get shared data of a policy.
+   *
+   * @param {string} policyId
+   * @return {!Promise<Object>}
+   */
+  getMergedSharedData(policyId) {
+    return this.whenPolicyResolved(policyId)
+        .then(() => this.ConsentStateManagerPromise_)
+        .then(manager => {
+          const promises = this.instances_[policyId].getConsentInstanceIds()
+              .map(consentId =>
+                manager.getConsentInstanceSharedData(consentId));
+          return Promise.all(promises);
+        }).then(sharedDatas => {
+          // preprend an empty object
+          // since Object.assign does not accept null as first argument
+          sharedDatas.unshift({});
+          return Object.assign.apply(null, sharedDatas);
```

Comment:
don't really care, but please comment that key pairs in sharedData may be overwritten after merge. We can come back to this once multi consents is supported.

Reply:
expaned the js doc to talk about how the data gets merged

Reply:
Do you plan to ask individual ad network to add their sharedData info to doc?

Reply:
yes. document will come later.